modules/tensor_mechanics/src/materials/ComputeMultipleInelasticCosseratStress.C

```diff
@@ -35,7 +35,8 @@ ComputeMultipleInelasticCosseratStress::ComputeMultipleInelasticCosseratStress(
 void
 ComputeMultipleInelasticCosseratStress::initQpStatefulProperties()
 {
-  ComputeMultipleInelasticStress::initQpStatefulProperties();
```

Comment:
Please could you make ```ComputeMultipleInelasticCosseratStress::initQpStatefulProperties()``` call ```ComputeMultipleInelasticStress::initQpStatefulProperties()```, as then when changes get made in the latter (such as this PR!) they are automatically picked up in the Cosserat case.